.analysis_options

```diff
@@ -0,0 +1,2 @@
+analyzer:
```

Comment:
Let's revert this for now. It's broken on the stable channel. I have a separate PR to add this. Will wait until the stable channel catches up.